AVsitter2/Makefile

```diff
@@ -56,7 +56,8 @@ OPTIMIZED=[AV]sitA.lslo\
 UNOPTIMIZED=Plugins/AVcontrol/LockGuard/[AV]LockGuard.lsl\
  Plugins/AVcontrol/LockGuard/[AV]LockGuard-object.lsl\
  Utilities/AVpos-shifter.lsl\
- Utilities/Noob-detector.lsl
+ Utilities/Noob-detector.lsl\
+ Plugins/AVvip/[AV]vip.lsl
```

Comment:
This line should be with the other plugins, two lines above.

Reply:
Fixed in https://github.com/AVsitter/AVsitter/pull/115/commits/b9862850ba9df1a74ac1385c614ab19a5bf4169d